History.md

```diff
@@ -4,6 +4,7 @@
 
 ### Fixes
 * StandardFilter: Fix missing @context on iterations (#1525) [Thierry Joyal]
+* Consolidate contextualization of inputs via the context (#XXX) [Thierry Joyal]
```

Comment:
Due to the potential "breaking change" in `StandardFilters` and `VariableLookup` where `Proc` are now properly handled, we might to look at this to be similar to #1527 flagged as deprecation for 6.0.

I do not think anyone expected to have `Proc` to be exposed directly to templates.

For standard filters specifically, that `respond_to?(:to_liquid)` might have left non-liquid aware object in by potential design error. I could see use cases leveraging this unexpectedly. They have an easy path forward to migrate, but a deprecation warning might be of interest here.

Reply:
"contextualization" is a new concept, which makes this change log entry very unclear about what is being fixed.  That seems like just a symptom of the fact that this PR combines multiple changes, a new feature (added method), behaviour fixes and some refactoring (which doesn't need to be in the changelog).